docs/stress-test.md

```diff
@@ -122,3 +141,128 @@ platform is deployment-specific but the following are some examples of dashboard
   memory usage remains high even after the purge starts because the hub aggressively caches DB state in
   memory and is apparently not cleaning up the cache references even after spawners and users are deleted
   from the database.
+
+<a name="profiling"></a>
+## Profiling
+
+During the stress test `py-spy` can be used to capture profiling data. It will generate SVGs
```

Comment:
```suggestion
During stress testing, or even normal operations, [py-spy](https://github.com/benfred/py-spy) can be used to capture profiling data. It will generate SVGs
```